server/routes/bookedtoday/arrivals/searchforexisting/multipleExistingRecordsFoundController.test.ts

```diff
@@ -41,7 +43,7 @@ beforeEach(() => {
   expectedArrivalsService.getMatchingRecords.mockResolvedValue(potentialMatches)
   expectedArrivalsService.getPrisonerDetails.mockResolvedValue(potentialMatches[0])
   flashProvider.mockReturnValue([])
-  signedCookiesProvider.mockReturnValue({ 'search-details': searchDetails })
+  stubCookie(State.searchDetails, searchDetails)
```

Comment:
this now has type safety - searchDetails needs to have same type as `SearchDetails`